backend/test/test_job.py

```
@@ -0,0 +1,156 @@
+from models.agent_model import ChangeType, JobStatus, RoleType
+from datetime import datetime
+from models.agent_model import JobStatusResponse
+from services.job import job_results
+
+def create_test_job(job_id: str):
+    job_results[job_id] = JobStatusResponse(
+        job_id=job_id,
+        status=JobStatus.QUEUED,
+        messages=[],
+        file_changes=[],
+        created_at=datetime.now().isoformat()
```

Ruff (0.15.11)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)